app/src/main/java/com/example/composetemplate/presentation/common/LectureCard.kt

```diff
@@ -0,0 +1,144 @@
+
+            if (!isExpanded) {
+                Row(
+                    verticalAlignment = Alignment.CenterVertically
+                ) {
+                    Image(
+                        painter = painterResource(id = R.drawable.ic_launcher_background), // Replace with your image
+                        contentDescription = null,
+                        modifier = Modifier
+                            .clip(CircleShape)
+                            .size(50.dp)
+                            .border(1.dp, CustomTheme.colors.circleImageBorder, CircleShape),
+                        contentScale = ContentScale.Crop
+                    )
+
+                    Text(
+                        text = lecture.author,
+                        style = CustomTheme.typography.getLectureAuthorStyle(),
+                        modifier = Modifier.padding(start = 16.dp)
+                    )
+                }
+            } else {
+                Column(
+                    horizontalAlignment = Alignment.CenterHorizontally,
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(8.dp)
+                ) {
+                    Image(
+                        painter = painterResource(id = R.drawable.ic_launcher_background), // Replace with your image
+                        contentDescription = null,
+                        modifier = Modifier
+                            .clip(CircleShape)
+                            .size(100.dp)
+                            .align(Alignment.CenterHorizontally)
+                            .border(1.dp, CustomTheme.colors.circleImageBorder, CircleShape),
+                        contentScale = ContentScale.Crop
+                    )
+
+                    Text(
+                        text = lecture.author,
+                        color = CustomTheme.colors.text,
+                        style = CustomTheme.typography.getLectureAuthorStyle(),
+                        modifier = Modifier.padding(top = 12.dp)
+                    )
+                }
+            }
+
+            Spacer(modifier = modifier.height(10.dp))
+
+            Text(
+                text = lecture.title,
+                color = CustomTheme.colors.text,
+                style = CustomTheme.typography.getLectureTopicStyle(),
+                modifier = Modifier.padding(start = 8.dp)
+            )
+
+            Spacer(modifier = modifier.height(6.dp))
+
+            Text(
+                text = if (lecture.isPast) LECTURE_CARD_PAST_TIME else DateUtil.lectureFormatDate(lecture.timeStamp),
+                style = CustomTheme.typography.getLectureTimeStampStyle(lecture.isPast),
+                modifier = Modifier.padding(start = 8.dp)
+            )
+        }
+    }
+}
+
+@Preview(showBackground = true)
+@Preview(showBackground = true, uiMode = Configuration.UI_MODE_NIGHT_YES)
+@Composable
+private fun LectureCardPrev() {
+    LectureCard(
```

Comment:
Maybe add a comment that this is an example, and later we will retrieve it from the server.